src/main/java/operation/impl/Purchase.java

```diff
@@ -0,0 +1,13 @@
+package operation.impl;
+
+import db.Storage;
+import model.FruitTransaction;
+import operation.OperationHandler;
+
+public class Purchase implements OperationHandler {
```

Comment:
Such a name makes it clear what the class is responsible for
refactor it everywhere
```suggestion
public class PurchaseOperationHandlerImpl implements OperationHandler {
```